src/pages/index.hbs

```diff
@@ -1,6 +1,6 @@
 ---
 layout: src/templates/layouts/landing.hbs
-title: An Invitation to Joy
+title: Ceaseless Prayer - An Invitation to Joy
```

Comment:
Should we even have "An Invitation to Joy?"
Should we be consistent? "Personal prayer for everyone on earth?"
Something else?